backend/api/router/router.go

```diff
@@ -159,6 +159,48 @@ func SetupRoutes(i *do.Injector, app *gin.Engine) error {
 		return err
 	}
 
+	// likePost /posts/{postId}/like POST
+	app.POST("/posts/:postid/like", postController.LikePost)
+	likePostOpe, _ := appDoc.NewOperationContext(http.MethodPost, "posts/{postId}/like")
+	likePostOpe.SetID("likePost")
+	likePostOpe.AddReqStructure(new(schema.LikePostRequest))
+	likePostOpe.SetSummary("対象の投稿をlikeする")
+	likePostOpe.SetTags("post")
+	likePostOpe.AddRespStructure(new(schema.MutationSchema), openapi.WithHTTPStatus(http.StatusCreated))
+	likePostOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusBadRequest))
+	likePostOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusForbidden))
+	likePostOpe.AddRespStructure(new(schema.ErrorResponse), openapi.WithHTTPStatus(http.StatusInternalServerError))
+	if err := appDoc.AddOperation(likePostOpe); err != nil {
+		return err
+	}
+	// getlikeRecord /posts/{postId}/getLikes POST
+	app.GET("/posts/:postid/getLikes", postController.GetLikeRecords)
+	getLikeRecordsOpe, _ := appDoc.NewOperationContext(http.MethodPost, "posts/{postId}/getLikes")
```

Comment:
単純にGET likeでも良いかなあと思いました
あと一部GETがPOSTになってます！
```suggestion
	// getlikeRecord /posts/{postId}/like GET
	app.GET("/posts/:postid/like", postController.GetLikeRecords)
	getLikeRecordsOpe, _ := appDoc.NewOperationContext(http.MethodGet, "posts/{postId}/like")
```